tensorrt_llm/llmapi/llm_args.py

```diff
@@ -1953,21 +1953,20 @@ def update_llm_args_with_extra_dict(
         "enable_build_cache": BuildCacheConfig,
         "speculative_config": DecodingBaseConfig,
         "lora_config": LoraConfig,
+        "cuda_graph_config": CudaGraphConfig,
```

Comment:
This is no longer necessary, as all Pydantic configurations can be parsed automatically. Only the non-Pydantic classes need to do explicit parsing here.

Reply:
Removed.